src/main/java/de/tum/in/www1/artemis/web/rest/TextAssessmentResource.java

```diff
@@ -137,8 +136,7 @@ public TextAssessmentResource(AuthorizationCheckService authCheckService, TextAs
     }
 
     /**
-     * DELETE text-submissions/:exampleSubmissionId/example-assessment : delete result & text blocks for example submission.
-     * This is used when updating the text of the example assessment.
+     * DELETE text-submissions/:exampleSubmissionId/example-assessment/feedback : delete feedback for example submission.
```

Comment:
the URL below does not make fully sense in my opinion, an example submission is not the same as a text submission.
Also the part `feedback` does not really make sense. Please change it as well so that we get this whole resource in a consistent state